### PR TITLE
Richard Flores HW01

### PR DESCRIPTION
**Pull request by RSFlores**
## Reflections
In this section, I describe how I got the changes into README.md before turning to reflect on my GitHub workflow and Markdown experience. I was able learn the fundamental for Markdown syntax from the GitHub [Guide](https://guides.github.com/features/mastering-markdown/) to mastering Markdown. The names of the journal titles are in italic with the use of a single asterisk at the beginning and end of the titles. The Repository for Homework 01 header is in bold with the use of two asterisks placed before Repository and after 01. 
I also learned how to add links with the github url (before the introduction) and the Github Guide to Markdown url (earlier in this section). These were accomplished by putting brackets between the word(s) that I wanted to place the url on. Then, I put the actual url in between parenthesis after the closed bracket. I learned how to use headers, which involved inserting the corresponding number of # for the tier of header that you want before the text itself. Lastly, I made an ordered list by following the proper spacing and number ordering (1,2) in my brief description of ongoing work.

### GitHub
My GitHub workflow follows the same principles outlined in the [guidelines](https://cfss.uchicago.edu/faq/homework-guidelines/#homework-workflow). First, I forked the repository from the class page. After cloning the repo to my laptop, I've worked from RStudio to modify the README.md. I will then commit the changes locally,before pushing them up to my GitHub page, and create a pull request. Before pushing changes, it is important to pull first so that you don't accidentally override anything that's been put in the repo since your last pull.

### Markdown
I have been pleasantly surprised with Markdown so far. The learning curve is not as steep as I thought, and it's nice to begin learning a text formatting language that is an important and relevant part of modern research and more general collaboration efforts. I wish there was an active spell check, though (but this might be strictly preferable given that Word all too often autocorrects text incorrectly).

### CODE REVIEW

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
Congratulations on the publications! Richard's bio demonstrates a good application of markdown grammar through the bold, list, link, and headlines. 
* Some specific praise?
Richard's reflection was really detailed and informative. 
* Something I learned?
Yikes! I did not realize there is no spell checker in RStudio until now. I am sure I have made quite a few spelling mistakes in my bio. 
* Specific constructive criticism?
1. For the listing in markdown, you don't need to add order yourself. Instead, you could just put all listing as "1." and markdown would automatically convert them into an ordered list. That is one less thing to keep track of (Yay!)
1. When committing changes to your README.md, I think it might be helpful to write a short comment about what change you've made in the commit in case you want to roll back to ealier commits. 
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
No really, Richard seems to find solutions to his questions in the assignment.

**Review by iradner — COMMENTED** [review]
## Peer Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |     x      |              |            |

Remarks:

This code was clean and reproducible and it included all the necessary elements. It might have been nice to add a pic. You can do this by using this code: _optional caption text_

It seems like Richard has a handle on the workflow and the use of git/github. I found this link useful in addition to the instructions on the class page: http://r-pkgs.had.co.nz/git.html
I noticed that Richard had multiple commits. One thing that might be good practice for the future is including a brief description of what was updated in the commit name. This helps to keep track of the changes made in each commit. 

Richard's pull request comment was clear and informative. Like the other peer reviewer said, it didn't cross my mind that there was no spell check (and I did indeed have a typo or two in my bio because of it!), so thanks for pointing that out Richard!

**Review by emezra — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:
Everything here looks great! You incorporated the four elements, and it looks smooth after viewing everything on my laptop. I also did not realize that there was a spellchecker in R. It seems like Richard did everything the way it was supposed to be done with the github workflow. 
.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
 # hw01
-Repository for homework 01
+**Repository for Homework 01**
+
+This is the repository of Richard Flores. I am excited to learn the ropes on [Github!](http://github.com)
+
+## Introduction
+I am a sixth year student in the doctoral program in political science here at UC. Broadly speaking, my research interests focus on American politics. More specifically, I'm motivated by understanding how intragroup dynamics drive public opinion formation of Latinxs. To that end, my dissertation project is working to assess the consequences that emotions have in the making of Latinx identity and their subsequent linkages to patterns of policy preferences and vote choice.
```
Comment (bensoltoff) [inline README.md:7]: Best practice is to leave a blank line between different types of Markdown content. For example, leave a blank line between a header and a block of text. It makes it easier to read in the original `.md` file, plus prevents certain kinds of rendering errors when you try to convert the file to HTML or PDF.

```
## Introduction

I am a sixth year student in the doctoral program in political science here at UC. Broadly speaking, my research interests focus on American politics. More specifically, I'm motivated by understanding how intragroup dynamics drive public opinion formation of Latinxs. To that end, my dissertation project is working to assess the consequences that emotions have in the making of Latinx identity and their subsequent linkages to patterns of policy preferences and vote choice. 
```

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |    x         |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |           |    x          |            |
| **Achievement, creativity** |       x    |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* It doesn't check as you type, but RStudio does have a [built-in spellchecker.](https://support.rstudio.com/hc/en-us/articles/200551916-Spelling-Dictionaries) Type F7 to run a spell check on an R Markdown/Markdown document. It's not always the best (for instance, it tries to spell check URLs as words), but is decently helpful.

**Pull request closed without merging**